### PR TITLE
[Feature][Zeta][REST-API]add get running job info by jobId

### PR DESCRIPTION
**Pull request by liugddx**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/incubator-seatunnel/blob/dev/release-note.md).

### CODE REVIEW

File: seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestHttpGetCommandProcessor.java [hunk 1]
```diff
@@ -67,10 +70,19 @@ public RestHttpGetCommandProcessor(
     @Override
     public void handle(HttpGetCommand httpGetCommand) {
         String uri = httpGetCommand.getURI();
-        if (uri.startsWith("/hazelcast/rest/maps/running-jobs")) {
-            handleRunningJobsInfo(httpGetCommand);
-        } else {
-            original.handle(httpGetCommand);
+        try {
+            if (uri.startsWith("/hazelcast/rest/maps/running-jobs")) {
```
Comment (Hisoka-X) [inline seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/rest/RestHttpGetCommandProcessor.java]: Maybe use constant string to manage url path maybe better.

**Comment by Hisoka-X** [comment]
Please add e2e test case for this change.

**Comment by TyrantLucifer** [comment]
> Please add e2e test case for this change.

Yup, add e2e test case is better.

**Comment by liugddx** [comment]
> > Please add e2e test case for this change.
> 
> Yup, add e2e test case is better.

Ok, I will provide e2e.

File: docs/en/seatunnel-engine/rest-api.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+### Returns an overview over all jobs and their current state.
+
+<details>
+ <summary><code>GET</code> <code><b>/hazelcast/rest/maps/running-jobs</b></code> <code>(Returns an overview over all jobs and their current state.)</code></summary>
+
+#### Parameters
+
+```json
+{}
```
Comment (Hisoka-X) [inline docs/en/seatunnel-engine/rest-api.md]: If requestbody is empty, should remove it.

Comment (liugddx) [reply docs/en/seatunnel-engine/rest-api.md]: I agree with you.

File: docs/en/seatunnel-engine/rest-api.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+## API reference
+
+### Returns an overview over all jobs and their current state.
+
+<details>
+ <summary><code>GET</code> <code><b>/hazelcast/rest/maps/running-jobs</b></code> <code>(Returns an overview over all jobs and their current state.)</code></summary>
+
+#### Parameters
+
+-
```
Comment (Hisoka-X) [inline docs/en/seatunnel-engine/rest-api.md]:
```suggestion

```

File: seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/RestApiIT.java [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+public class RestApiIT {
+
+    private static final String HOST = "http://localhost:";
+
+    private static ClientJobProxy clientJobProxy;
+
+    private static HazelcastInstanceImpl hazelcastInstance;
+
+    @BeforeAll
+    static void beforeClass() throws Exception {
```
Comment (EricJoy2048) [inline seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/RestApiIT.java:57]: Add @AfterAll and cancel the job and shutdown the cluster?

Comment (liugddx) [reply seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/RestApiIT.java:57]: > Add @afterall and cancel the job and shutdown the cluster?

Done.

**Review by EricJoy2048 — APPROVED** [review]

**Review by Hisoka-X — APPROVED** [review]

**Pull request merged**